src/test/java/net/snowflake/ingest/SimpleIngestIT.java

```diff
@@ -331,9 +333,18 @@ private void verifyDefaultUserAgent(
 
   @Ignore
   @Test
-  public void testConfigureClientHappyCase() throws Exception {
+  public void testConfigureClientHappyCaseNullRole() throws Exception {
     final String userAgentSuffix = "kafka-provider/NONE";
-    SimpleIngestManager manager = TestUtils.getManager(pipeName, userAgentSuffix);
+    SimpleIngestManager manager = TestUtils.getManager(pipeName, userAgentSuffix, null);
+    ConfigureClientResponse configureClientResponse = manager.configureClient(null);
+    assertEquals(0L, configureClientResponse.getClientSequencer().longValue());
+  }
+
+  @Ignore
```

Comment:
why these tests are ignored for now? Could you add a JIRA for enabling them?

Reply:
These tests are disabled from the beginning as they arose. I don't know what the story is behind this.
https://github.com/snowflakedb/snowflake-ingest-java/pull/105/files

Reply:
@sfc-gh-japatel i think it could be enabled now?

Reply:
I have it enabled already but I forgot to push.